### ISSUE TITLE
Include in the DSC Resource Kit

### ISSUE THREAD

**Issue by jonathaneckman** [jcwalker/xAdfs#4]
Add this module to the [DSC Resource Kit](https://github.com/PowerShell/DscResources).

Prerequisites referenced [here](https://github.com/PowerShell/DscResources/issues/417).

**Comment by X-Guardian** [comment]
Hi @jcwalker, it would be great if this module was made part of the DSC Resource Kit and published to the PowerShell Gallery. I'm happy to submit some PR's to get this up to the [HQRM guidelines](https://github.com/PowerShell/DscResources/blob/master/HighQualityModuleGuidelines.md), if you are available to review them?

**Comment by jcwalker** [comment]
I don't think any new resources are allowed in the DSC resource kit.  However, if you are willing to make the changes to put it inline with the HQRM guidelines we can definitely publish it in the gallery.

**Comment by X-Guardian** [comment]
Yes they are. See [New Resource Module Submissions](https://github.com/PowerShell/DscResources/blob/master/NewResourceModuleSubmissions.md).
I'll start raising PR's to bring the module in line with the guidelines.

**Comment by jcwalker** [comment]
That page references adding a submodule to an existing module.  I say they aren't accepting any new modules because a colleague tried to get AccessControlDsc included in the resource kit and was told no new resources in the resource kit.

**Comment by X-Guardian** [comment]
Hi @johlju, can modules still be added to the DSC resource kit?

**Comment by johlju** [comment]
Not added, but linked from the DSC Resource Kit. We could always discuss adding it to DSC Community if that would be of interest.

**Comment by X-Guardian** [comment]
Ok, the same as xWebAdministration etc, yes?

**Comment by johlju** [comment]
xWebAdministration is in the PowerShell organization currently, but hopefully it will be moved together with the rest of the DSC repos to DSC community organization. 
https://GitHub.com/dsccommunity
https://dsccommunity.org

**Comment by johlju** [comment]
If linked to the DSC Recource Kit in the PowerShell organization the repo would still be at this current location.

**Comment by johlju** [comment]
For the later, read more here https://github.com/PowerShell/DscResources/blob/master/NewResourceModuleSubmissions.md

**Comment by X-Guardian** [comment]
Cheers @johlju.

@jcwalker, I've got an updated branch with a preliminary  working AppVeyor build. Can you create a dev branch on this Repo and make it the default, then add the Repo to Appveyor. I'll then raise a PR from my branch to dev.

**Comment by jcwalker** [comment]
@X-Guardian when I attempt to transfer the repository to you I receive the message "X-Guardian\xAdfs" already exists.  I'm not sure the best course of action is.  Maybe rename your current fork of xAdfs is that's possible?

**Comment by X-Guardian** [comment]
I've deleted my fork of this repo, so you should be able to transfer it now.
